pkgs/development/cuda-modules/cutensor/extension.nix

```diff
@@ -96,11 +97,14 @@ let
   redistArch = flags.getRedistArch hostPlatform.system;
   # platformIsSupported :: Manifests -> Boolean
   platformIsSupported =
-    { feature, ... }:
+    { feature, redistrib, ... }:
     (attrsets.attrByPath [
       pname
       redistArch
-    ] null feature) != null;
+    ] null feature) != null
+
+    # NOTE: This is an ad hoc hack; manifest schemas do not support version constraints yet
+    && !(lib.versionOlder cudaVersion "11.0" && lib.versionAtLeast redistrib.${pname}.version "2.0.2");
```

Comment:
@ConnorBaker I don't suppose it's worth the effort trying to accommodate this generically until after we've refactored cuda to have only one evalModules call

Reply:
Btw older versions of cutensor are apparently broken too: https://hercules-ci.com/accounts/github/SomeoneSerge/derivations/%2Fnix%2Fstore%2Fk4s2k2ilxf4h5rj70jm13ip7kzbyr3pl-libcutensor-1.5.0.3.drv/log?via-job=6bd1847f-fca5-4a9a-b3d4-4fac2eeba5a0